src/Forms/HTMLEditor/TinyMCECombinedGenerator.php

```diff
@@ -9,10 +9,12 @@
 use SilverStripe\Core\Flushable;
 use SilverStripe\Core\Injector\Injectable;
 use SilverStripe\Core\Manifest\ModuleResource;
+use SilverStripe\Dev\Deprecation;
 
 /**
  * Generates tinymce config using a combined file generated via a standard
  * SilverStripe {@link GeneratedAssetHandler}
+ * @deprecated 5.4.0 Will be replaced with SilverStripe\TinyMCE\TinyMCECombinedGenerator
```

Comment:
I've recommended we change the namespace in the new module to from `\TinyMCE\` to `\HtmlEditorTinyMCE\` [here](https://github.com/silverstripe/silverstripe-htmleditor-tinymce/pull/1#discussion_r1988325388) 

Applies to any other references to the namespace in this PR